### ISSUE TITLE
Warning: PDOStatement::execute(): SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens

### ISSUE THREAD

**Issue by myleshk** [waza-ari/monolog-mysql#23]
### Error message:
```
Warning: PDOStatement::execute(): SQLSTATE[HY093]: Invalid parameter number: number of bound variables does not match number of tokens in **/vendor/wazaari/monolog-mysql/src/MySQLHandler/MySQLHandler.php on line 214
```

### Situation:
When some `additionalFields` are provided with empty values.
No logs are recorded to database when this warning shows.

### Code problem observed:
While `$contentArray` contains the full list of key-values,
`$this->statement` is only prepared with keys that has non-empty values. Thus it is true that ` number of bound variables does not match number of tokens`.

**Comment by darkoromanov** [comment]
I've added this code at line 208:

```
//Remove unused keys
foreach($this->additionalFields as $key => $context) {
    if(! isset($contentArray[$key]))
        unset($this->additionalFields[$key]);
}
```

**Comment by blancessanchez** [comment]
test

**Comment by cherryred5959** [comment]
Fixed #23
